### PR TITLE
Adds code quality badges to repo for flapjack branch

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds TravisCI, Coveralls, and Code Climate code quality badges to repo for flapjack branch.

## Additions

- Adds TravisCI, Coveralls, and Code Climate code quality badges to repo for flapjack branch in README.md.

## Testing

- Check URLs for badges.

## Review

- @KimberlyMunoz 
- @sebworks 
- @jimmynotjim 

## Screenshots

screen shot 2015-08-25 at 2 38 07 pm

### CODE REVIEW

**Comment by jimmynotjim** [comment]
What do we have to do to get Coveralls out of unknown?

**Comment by anselmbradford** [comment]
> What do we have to do to get Coveralls out of unknown?

Not sure, I was hoping adding the badge might be enough. The coveralls page currently gives a warning that the repo isn't badged.

**Comment by jimmynotjim** [comment]
Ah, I should have read that part. Looks like once it's merged to flapjack branch. It looks like it uses the default, so you can remove that passed branch.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # cfgov-refresh
 
+[![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
+[![Coverage Status](https://coveralls.io/repos/cfpb/cfgov-refresh/badge.png?branch=flapjack)](https://coveralls.io/r/cfpb/cfgov-refresh?branch=beta)
```
Comment (KimberlyMunoz) [inline README.md]: This links to beta when the badge is for flapjack.

Comment (anselmbradford) [inline README.md]: Oops, thanks! Will fix

Comment (anselmbradford) [inline README.md]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/a386709db2a7edff603b2999ebabacbbefb8b4a2 Also updated badge image URL to that shown in code climate example.

**Comment by jimmynotjim** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**